metadata-ingestion/docs/sources/kafka-connect/kafka-connect.md

```diff
@@ -1,5 +1,60 @@
 ## Advanced Configurations
 
+### Working with Platform Instances
+If you've multiple instances of kafka OR source/sink systems that are referred in your kafka-connect, you'd need to configure platform instance for these systems in kafka-connect recipe to generate correct lineage edges. You must have already set `platform_instance` in recipes of original source/sink systems. Refer the document [Working with Platform Instances](https://datahubproject.io/docs/platform-instances) to understand more about this.
```

Comment:
Let's please change kafka-connect -> `kafka-connect` to make this easier to read.

Reply:
done